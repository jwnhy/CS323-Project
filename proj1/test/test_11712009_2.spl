int token_and_coruppted_hex(int a, int b)
{
    int ? = 0123;
    int $ = 0x01234;
    int @ = '\x02ggg';
    while(){}
    if(){}
}