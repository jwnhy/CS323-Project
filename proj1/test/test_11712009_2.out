Error type A at Line 1: Invalid Character, ?
Error type A at Line 1: Leading Zero Inside Literal, 0123
Error type A at Line 1: Invalid Character, $
Error type A at Line 1: Leading Zero Inside Literal, 0x01234
Error type A at Line 1: Invalid Character, @
Error type A at Line 1: Invalid Hex Char, '\x02ggg'
Error type B at Line 1: Missing Expression.
Error type B at Line 1: Missing Expression.
